src/platform/silabs/efr32/efr32-chip-mbedtls-config.h

```diff
@@ -34,14 +34,20 @@
 
 // MATTER AWS Specific Configurations
 #ifdef SL_MATTER_ENABLE_AWS
+#define MBEDTLS_DEBUG_C
```

Comment:
- Why is this DEBUG related macro defined? Is this intentional or missed removing this? If intentional, how will it be used?